streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/BaseJoinProcessorNode.java

```diff
@@ -63,10 +63,6 @@
         return joinMergeProcessorParameters;
     }
 
-    ValueJoinerWithKey<? super K, ? super V1, ? super V2, ? extends VR> valueJoiner() {
```

Comment:
As it is an internal class and this method is not use in the code base, I assume is safe to delete.